src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -801,6 +802,7 @@ export async function install(
     modOptions
   }: InstallArgs
 ): Promise<InstallResult> {
+  console.log('installing game with siwe values ', siweValues)
```

Comment:
Just making a note of the comment here

Reply:
definitely a debug log I forgot that I left in. just removed it